app/views/shared/_component_environments.erb

```diff
@@ -1,9 +1,11 @@
 <div class="govuk-form-group <%= 'govuk-form-group--error' if @component.errors.key?(:environment) %>">
   <h3 class="govuk-heading-m"><%= t 'components.environment' %></h3>
   <div class="govuk-radios">
-    <div class="govuk-radios__item ">
-      <%= f.radio_button :environment, ENVIRONMENT::STAGING, class: 'govuk-radios__input'  %>
-      <%= f.label :environment, ENVIRONMENT::STAGING.capitalize, class: 'govuk-label govuk-radios__label' %>
-    </div>
+    <% @hub_environments&.each do |hub_environment| %>
```

Comment:
maybe?
```suggestion
    <% @hub_environments.each do |hub_environment| %>
```

also, can we update the model validation to accept only these environments? currently it only validates the presence -> https://github.com/alphagov/verify-self-service/blob/b46af1b73cb6583ed0b9c5e5203d86fa4bb6eec3/app/models/new_msa_component_event.rb#L7

(components **must** have environment)

Reply:
I love &.

Reply:
I know you do! :D ... it's good when the thing is optional, but in here it's required...